packages/portfolio-contract/test/portfolio.contract.test.ts

```diff
@@ -355,3 +355,96 @@ test('open portfolio with USDN, Aave positions', async t => {
   t.snapshot(contents, 'vstorage');
   t.snapshot(done.payouts, 'refund payouts');
 });
+
+test.only('open portfolio with USDN position and withdraw funds', async t => {
+  const { trader1, common } = await setupTrader(t);
+  const { usdc, poc24 } = common.brands;
+
+  // First, open a portfolio with USDN position
+  const doneP = trader1.openPortfolio(
+    t,
+    {
+      USDN: usdc.units(3_333),
+      NobleFees: usdc.make(100n),
+      Access: poc24.make(1n),
+    },
+    { destinationEVMChain: 'Ethereum' },
```

Comment:
We shouldn't need to specify an EVM chain to open a USDN position. I **suggest** an `// XXX ` comment